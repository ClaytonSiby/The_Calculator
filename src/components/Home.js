import React from 'react'

const Home = () => (
  <div className='homeContainer'>
    <div>
      <p>
        Mathematics as an expression of the human mind reflects the active will,
        the contemplative reason, and the desire for aesthetic perfection. Its
        basic elements are logic and intuition, analysis and construction,
        generality and individuality.
      </p>
      <p>
        We will always have STEM with us. Some things will drop out of the
        public eye and go away, but there will always be science, engineering
        and technology. And there will always, always be mathematics.
      </p>
    </div>
  </div>
)

export default Home
