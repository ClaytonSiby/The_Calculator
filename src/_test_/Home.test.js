import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('Home component', () => {
  test('renders the DOM without errors', () => {
    const test = renderer.create(<Home />).toJSON();
    expect(test).toMatchSnapshot();
  });
});
