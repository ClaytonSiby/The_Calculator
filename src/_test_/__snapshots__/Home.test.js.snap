// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders the DOM without errors 1`] = `
<div
  className="homeContainer"
>
  <div>
    <p>
      Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality.
    </p>
    <p>
      We will always have STEM with us. Some things will drop out of the public eye and go away, but there will always be science, engineering and technology. And there will always, always be mathematics.
    </p>
  </div>
</div>
`;
